package.json

```diff
@@ -2,10 +2,12 @@
   "name": "apimocking",
   "version": "1.0.0",
   "description": "An tool to test communication between your microservices",
-  "main": "main.js",
+  "main": "public/main.js",
   "scripts": {
     "test": "npm test",
-    "start": "electron ."
+    "start": "electron .",
+    "build": "webpack",
+    "eDev": "webpack-dev-server --hot --config webpack.config.js --mode development"
```

Comment:
Can you clarify when we should be using `electron` vs `build` vs `eDev`? Might help to have that documented somewhere so everyone’s on the same page.

Reply:
We will be using the eDev, when the dev server is fully functional. for now until its fixed, the flow will be, make a change, run npm build, then npm start.  In another process we will  discuss the new flow.